base/steer/FairRunAnaProof.h

```diff
@@ -82,6 +82,7 @@ class FairRunAnaProof : public FairRunAna
     FairRunAnaProof operator=(const FairRunAnaProof&);
 
     FairFileSource* fProofFileSource;
+    FairEventHeader* fEvtHeader = nullptr;
```

Comment:
Which code in FairRunAnaProof initalizes it to the real pointer?

Reply:
Initialisation of event header is in `FairRun`. And pointer `fEvtHeader` in `FairRunAnaProof` is not onwed.

Reply:
Oh, it seems I misunderstood the "initialisation" of the pointer. If it means assigning a valid address to the pointer, ah, yes, only FairRunAnaProof can initialise it:
 https://github.com/FairRootGroup/FairRoot/blob/95b6b43de7ff33308989473c2cf90132effc3fa8/base/steer/FairRunAnaProof.cxx#L243

That's the only place when it's assigned. And in the `Init()` function the pointer cannot be guaranteed to have a valid memory address. I will make a change about it very soon. (also in other classes).